packages/office-ui-fabric-react/src/components/Panel/__snapshots__/Panel.test.tsx.snap

```diff
@@ -102,10 +102,6 @@ exports[`Panel renders Panel correctly 1`] = `
               {
                 -moz-osx-font-smoothing: grayscale;
                 -webkit-font-smoothing: antialiased;
-                animation-duration: 0.167s;
-                animation-fill-mode: both;
-                animation-name: keyframes from{opacity:0;}to{opacity:1;};
-                animation-timing-function: cubic-bezier(.1,.25,.75,.9);
```

Comment:
Why this animation is removed? Is it not breaking the animation?

Reply:
I believe that this is actually fixing a bug, a currently rendered panel should not have an animation on it. The code reads like the animation should only be added when it's animated in or out. The test as it's written just creates a panel that's already open, which to me means that it shouldn't animate . We can add some extra stuff to check for that. Animations definitely do work though.

Reply:
Very interesting that a change of the `isOpen` from being a state property to a class private property would change a screenshot in that manner, even if the explanation you gave makes perfect sense. Anyways I am glad that everything still works as expected.